numpy/core/__init__.py

```diff
@@ -153,13 +152,13 @@ def _DType_reduce(DType):
 
 
 def __getattr__(name):
-    # Deprecated 2021-10-20, NumPy 1.22
-    if name == "machar":
+    # Deprecated 2022-11-21, NumPy 1.24.
```

Comment:
Needs to be NumPY 1.25 now :)

Reply:
Hah, fine, maybe makes life easier for Numba (or it doesn't because for them the other one matters)...